### PR TITLE
MD-6148 : Manufacturers Dashboard changes

### PR DESCRIPTION
**Pull request by sunnyparspec21**

### CODE REVIEW

File: src/components/AutoComplete/Autocomplete.tsx [hunk 1]
```diff
@@ -22,20 +22,32 @@ export interface AutocompleteProps {
...
     defaultValue?: string | OptionType | (string | OptionType)[] | null;
     onBlur?: (params: OptionType | string) => void;
-    helperText?: string;
+    helperText?: string | React.ReactNode;
     error?: boolean;
     onTextFieldChange?: (e: React.SyntheticEvent<Element, Event>, value: string) => void;
     limitTags?: number;
+    disabled?: boolean;
+    clearOnBlur?: boolean;
+    filterOptionsCallBack?: (options: OptionType[], params: any) => OptionType[];
```
Comment (Charul-Parspec) [inline src/components/AutoComplete/Autocomplete.tsx:31]: @komalkiri define specific type for params(tip. hover over filterOptions prop in MuiAutocomplete, you will get the typ)

**Review by Charul-Parspec — CHANGES_REQUESTED** [review]

File: src/components/AutoComplete/Autocomplete.tsx (hunk 1 above)
Comment (komalkiri) [reply src/components/AutoComplete/Autocomplete.tsx]: Done

**Review by Charul-Parspec — APPROVED** [review]

**Review by sunnyparspec21 — COMMENTED** [review]
looks good to me

File: src/components/Table/Table.tsx [hunk 2]
```diff
@@ -559,6 +571,7 @@ Table.defaultProps = {
     searchSettings: {
         hierarchyMode: 'Both'
     },
-    hiddenProperty: 'is_hidden'
+    hiddenProperty: 'is_hidden',
+    selectedItemsBelowSearch: false
```
Comment (abhijit-parspec) [inline src/components/Table/Table.tsx:575]:
```suggestion
    showSelectedItemsCountBelowSearch: false
```

File: src/components/Table/Table.tsx [hunk 3]
```diff
@@ -456,6 +458,16 @@ export const Table: React.FC<TableProps> = forwardRef((props, ref) => {
                     <Box>{rightSection}</Box>
                 </Box>
             )}
+            {toolBarOptions?.includes('selectedItems') && selected > 0 && selectedItemsBelowSearch && (
+                <Box p={1} pl={3} pr={2} mb={2} bgcolor={'primary.main'} color={'secondary.contrastText'} display="flex" alignItems="center" gap={2} width={'max-content'}>
```
Comment (abhijit-parspec) [inline src/components/Table/Table.tsx:462]:
```suggestion
                SelectedItemsCount
```

**Review by abhijit-parspec — APPROVED** [review]

**Pull request merged**